api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -40,6 +42,7 @@ public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements O
 	@FullTextField(analyzer = SearchAnalysis.NAME_ANALYZER)
 	private String name;
 	
+	@Size(max = 255)
```

Comment:
I'm not really sure why we need this `@Size` annotation here? Isn't the length already covered by the `@Column` annotation.